src/main/java/filemanager/launcher/Launcher.java

```diff
@@ -3,13 +3,28 @@
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import filemanager.binder.FileServiceBinderModule;
-import filemanager.controller.Application;
+import filemanager.configuration.FileHandlerConfiguration;
+import filemanager.directorytracker.ScheduleFileDirectory;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Environment;
 
-public class Launcher {
-    public static void main(String[] args) {
-        Injector guice = Guice.createInjector(new FileServiceBinderModule());
-        Application application = guice.getInstance(Application.class);
-        application.runProgram();
+import java.util.concurrent.TimeUnit;
+
+public class Launcher extends Application<FileHandlerConfiguration> {
+    public static void main(String[] args) throws Exception {
+        new Launcher().run(args);
     }
 
+    @Override
+    public void run(FileHandlerConfiguration configuration, Environment environment) {
+        Injector guice = Guice.createInjector(new FileServiceBinderModule());
```

Comment:
Hint: here you can pass ```configuration``` into module constructor, save it in module property and use it in ```@Provides``` methods for instantiating dependencies.